eZ/Publish/Core/Repository/ObjectStateService.php

```diff
@@ -380,7 +380,7 @@ public function updateObjectState(APIObjectState $objectState, ObjectStateUpdate
             throw $e;
         }
 
-        return $this->buildDomainObjectStateObject($spiObjectState);
+        return $this->buildDomainObjectStateObject($spiObjectState, null, $objectState->prioritizedLanguages);
```

Comment:
Please add `@property-read string[] $prioritizedLanguages` to `\eZ\Publish\Core\Repository\Values\ObjectState\ObjectState`